src/main/java/org/opentripplanner/routing/core/SimpleIntersectionTraversalCostModel.java

```diff
@@ -101,12 +108,25 @@ private double computeCyclingTraversalCost(IntersectionVertex v, StreetEdge from
         var turnAngle = calculateTurnAngle(from, to, request);
```

Comment:
`RoutingRequest` also contains a field for the driving direction ([`driveOnRight`](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java#L589-L593)), which is used by `calculateTurnAngle()`. The code here seems to work independently of that.

Reply:
Well spotted! 

Am I reading it correctly that when you set the driving direction then to left hand side, then a left turn is magically turned into a right hand one? If this is the case then I don't need change anything.

Reply:
Ok, after a bit of investigation, this does exactly what we need. I've added tests to verify.